src/game_inventory.cpp

```diff
@@ -23,8 +23,17 @@ typedef std::function<bool( const item_location & )> item_location_filter;
 namespace
 {
 
-std::string good_bad_none( int value )
+std::string good_bad_none( int value, bool warning = false )
 {
+    if( warning ) {
```

Comment:
This can be extracted into a separate function. It would be much cleaner without the boolean flag.